src/linux/mod.rs

```diff
@@ -942,8 +942,8 @@ impl LinuxPackageReadout {
     /// Returns the number of installed packages for systems
     /// that utilize `nix` as their package manager.
     fn count_nix() -> Option<usize> {
-        // Return the number of installed packages using sqlite (~10ms)
-        // as directly calling nix is a bit more expensive (~40ms)
+        // Return the number of installed packages using nix's database (~10ms)
+        // as directly calling to nix is a bit more expensive (~40ms)
```

Comment:
The benchmarks you listed in the comments should suffice, we shouldn't really have these in the comments.